templates/thank-you.html

```diff
@@ -1,40 +1,60 @@
 {% extends 'base_layout.html' %}
 {% block title %}MicroK8s - Contact-us{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o/edit{% endblock meta_copydoc %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o/edit{% endblock
+meta_copydoc %}
 
 {% block content %}
 
-<section class="p-strip is-deep">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Thank you for contacting us about MicroK8s</h1>
-      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-    </div>
-    <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
-      <img src="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg" alt="" width="200" height="200">
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col"></div>
+      <div class="col">
```

Comment:
```suggestion
    <div class="row">
      <div class="col-start-large-4">
```
To avoid the use of empty elements

Reply:
Using numbered grid columns shifts it a bit to the left and misaligns with the header, already tried that at first

Reply:
@akbarkz sorry to grave dig but just fyi for the future, the above doesn't work because it's just an offset, you're not defining a col size and that's what breaks the grid. It would need to be `col-start-large-4 col-9`. More on offsets [here](https://vanillaframework.io/docs/patterns/grid#empty-columns)